src/globalState/provider/TopicsProvider.tsx

```diff
@@ -61,5 +61,13 @@ export function useTopics() {
 }
 
 export function useTopic(id?: number) {
-	return getTopics(useTopics(), id);
+	const topics = useTopics();
+
+	try {
+		return getTopics(topics, id);
+	} catch (e) {
+		console.error(e);
```

Comment:
Could this happen? Could you check all places where useTopic is used and check if optinal chaining is added?